tools/ci_build/build.py

```diff
@@ -776,9 +776,6 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
                                                      args.minimal_build or args.use_extensions))
                                                      else "OFF"),
         "-Donnxruntime_REDUCED_OPS_BUILD=" + ("ON" if is_reduced_ops_build(args) else "OFF"),
```

Comment:
This may have another issue, if we pre-generate the reduced op code files, then `onnxruntime_REDUCED_OPS_BUILD` may not be set to true anymore, so the src substitution will not happen